src/pages/Mypage/Components/Mainbox/History/Item.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useNavigate } from 'react-router-dom';
+import API from '../../../../../config';
+
+const Item = ({
+  title,
+  sub_title,
+  image,
+  location,
+  reservation,
+  running_date,
+  place_id,
+  setIsClicked,
+}) => {
+  const navigate = useNavigate();
+
+  const reservationDelete = () => {
+    fetch(`${API.History}`, {
+      headers: {
+        Authorization: localStorage.getItem('Authorization'),
+      },
+      method: 'POST',
+      body: JSON.stringify({
+        place_id: place_id,
```

Comment:
```suggestion
      body: JSON.stringify({
        place_id
```

* key,value가 같으면 이렇게도 줄일 수 있어요~

Reply:
아앗 감사합니다!